cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -435,7 +461,15 @@ static DataprocConf create(Map<String, String> properties, @Nullable SSHPublicKe
     int workerMemoryGB = getInt(properties, "workerMemoryMB", 15 * 1024);
 
     int masterDiskGB = getInt(properties, "masterDiskGB", 1000);
+    String masterDiskType = getString(properties, "masterDiskType");
+    if (masterDiskType == null) {
+      masterDiskType = "pd-standard";
+    }
     int workerDiskGB = getInt(properties, "workerDiskGB", 1000);
+    String workerDiskType = getString(properties, "workerDiskType");
+    if (workerDiskType == null) {
+      workerDiskType = "pd-ssd";
```

Comment:
Does this change the default from standard to ssd? Is there cost implication?

Reply:
oops this was unintentional, changing it to standard